Libraries/LibWeb/Infra/Types.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2024, Sam Atkins <sam@ladybird.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Types.h>
+#include <AK/UFixedBigInt.h>
+
+namespace Web::Infra {
+
+// https://infra.spec.whatwg.org/#booleans
+// A boolean is either true or false.
+using Boolean = bool;
+
+// https://infra.spec.whatwg.org/#8-bit-unsigned-integer
+// An 8-bit unsigned integer is an integer in the range 0 to 255 (0 to 28 − 1), inclusive.
+using Unsigned8BitInteger = u8;
+
+// https://infra.spec.whatwg.org/#16-bit-unsigned-integer
+// A 16-bit unsigned integer is an integer in the range 0 to 65535 (0 to 216 − 1), inclusive.
+using Unsigned16BitInteger = u16;
+
+// https://infra.spec.whatwg.org/#32-bit-unsigned-integer
+// A 32-bit unsigned integer is an integer in the range 0 to 4294967295 (0 to 232 − 1), inclusive.
+using Unsigned32BitInteger = u32;
+
+// https://infra.spec.whatwg.org/#64-bit-unsigned-integer
+// A 64-bit unsigned integer is an integer in the range 0 to 18446744073709551615 (0 to 264 − 1), inclusive.
```

Comment:
missing to the power of things like: 2**64 or 2^64 - unfortunately they don't copy paste :(

Reply:
Ah, well spotted! Thanks. :^) Fixed.